int64 action
---
bool result
